files/walker.go

```diff
@@ -7,6 +7,7 @@ import (
 	"os"
 	"os/user"
 	"path/filepath"
+    "runtime"
```

Comment:
gofmt pls

Reply:
spaces > tabs

Reply:
don't worry be happy https://github.com/keeferrourke/imgrep/commit/0a7ccd740124650edf9e495d9f56344b2cab781a